py/mattak/backends/uproot/dataset.py

```diff
@@ -312,7 +333,7 @@ def __calibrate(self, waveform_array : numpy.ndarray, param : numpy.ndarray,
 
             for s, (adc, p) in enumerate(zip(wf_channel, param_channel)):
                 # discrete inverse
-                adcsamples = numpy.polyval(p[::-1], vsamples)
+                adcsamples = numpy.polyval(p[::-1], vsamples) + res_interp[int(c/12)]
```

Comment:
`+ res_interp[int(c/12)](vsamples)` right?

Reply:
res_interp is just an array for the lookup tabel of adcsamples, the actual interpolation happens later but I'll rename it for clarity